plugins/modules/net_tools/haproxy.py

```diff
@@ -62,6 +62,18 @@
     type: str
     required: true
     choices: [ disabled, drain, enabled ]
+  agent:
+    description:
+      - Disable/enable agent checks (depending on I(state) value).
+    type: bool
+    default: no
+    version_added: 1.0.0
```

Comment:
```suggestion
    version_added: 1.0.0
```
@felixfontein shouldn't it be changed to `1.1.0`?

Reply:
Depends. When it is merged by Friday, 1.0.0 is correct.